app/controllers/posts_controller.rb

```diff
@@ -27,9 +27,26 @@ def update
       render 'edit'
     end
   end
-
+# # search
```

Comment:
Yes @babiracka . And if you notice the first commit I made after merging your PR was removing commented code and extra white spaces. Here (https://github.com/rishijain/oneplaceaday/commit/05b180bdbda15e8d452f9a9366974e5f361dc5a1)